README.md

```diff
@@ -14,16 +15,19 @@
   - [Running the tests](#running-the-tests)
   - [Running the linter](#running-the-linter)
   - [Running all checks](#running-all-checks)
-- [Infrastructure Tech Stack](#infrastructure-tech-stack)
-  - [Cloud Platform](#cloud-platform)
-  - [Kubernetes](#kubernetes)
-  - [Docker](#docker)
-  - [AWS Elastic Container Registry](#aws-elastic-container-registry)
-  - [CircleCI](#circleci)
-  - [AWS](#aws)
-- [Useful Commands](#useful-commands)
+- [Useful commands](#useful-commands)
   - [kubectl](#kubectl)
-  - [aws](#aws-1)
+  - [aws](#aws)
+
+## About this project
+
+A long-lived API that exposes data from the HMPPS systems such as NOMIS, nDelius and OASys by being a single entry point for consumers. It's built using [Spring Boot](https://spring.io/projects/spring-boot/) and [Kotlin](https://kotlinlang.org/) as well as the following technologies for its infrastructure:
```

Comment:
Couple of optional changes:

> A long-lived API that exposes data from the HMPPS systems 

Remove "the"

> nDelius and OASys by being a single entry point for consumers.

"...and OASYS providing a single point of entry for consumers."

Reply:
Much better!